src/issue-details.ts

```diff
@@ -0,0 +1,197 @@
+export interface IssueDetails {
+  id: string
+  number: number
+  title: string
+  body: string
+  state: string
+  created_at: string
+  updated_at: string
+  closed_at: string | null
+  comments: number
+  reactions: ReactionsInfo
```

Comment:
This probably should be `reactions: number` and then a new `reactions_data: ReactionData[]`